pkg/auth/audit/audit.go

```diff
@@ -280,11 +282,24 @@ func isExist(array []string, key string) bool {
 	return false
 }
 
+func redactedBodyError(auditErr error) []byte {
+	m := map[string]string{
+		auditLogErrKey: auditErr.Error(),
+	}
+
+	body, err := json.Marshal(m)
+	if err != nil {
+		logrus.Debugf("auditLog: failed to generate audit log error body: %v", err)
```

Comment:
Might want to consider actually logging as error here (and potentially casting as %s for readability)

Reply:
Looking at https://github.com/rancher/rancher/wiki/Rancher-Development-Best-Practices#be-mindful-of-error-logging I don't think this is quite important enough since I don't think users are going to be handling those.